docs/api.md

```diff
@@ -359,6 +359,13 @@ puppeteer.launch().then(async browser => {
 - `options` <[Object]>
   - `browserWSEndpoint` <[string]> a [browser websocket endpoint](#browserwsendpoint) to connect to.
   - `ignoreHTTPSErrors` <[boolean]> Whether to ignore HTTPS errors during navigation. Defaults to `false`.
+  - `defaultViewport` <?[Object]> . Defaults to an 800x600 viewport. `null` disables the default viewport.
```

Comment:
Nit: unneeded spaces and period in `<?[Object]> . Defaults`?

Reply:
Oops, accidentally deleted the sentence that was there. Good catch!